### PR TITLE
Added description for fil_actor_interface

### LINKED ISSUE #32: Prepare for publish
Todo list before publishing to crates.io
- [X] Update the package description of every crate to be ready for publishing
- [X] Some crates have alpha in their version.
- [X] Some shared dependencies are workspace dependencies and some are not.
- [X] Move fil_utils to a fil_actor_interface or somewhere better

### PR DESCRIPTION
**Pull request by sudo-shashank**
**Summary of changes**
Changes introduced in this pull request:
- Added missing description for all packages

**Reference issue to close (if applicable)**

Contributes to #32 

**Other information and links**

### CODE REVIEW

File: fil_actor_interface/Cargo.toml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [package]
 name = "fil_actor_interface"
+description = "Backward compatibility layer for all versions of builtin actor."
```
Comment (LesnyRumcajs) [inline fil_actor_interface/Cargo.toml:3]: Are those all versions or only starting from v8?

**Review by lemmih — CHANGES_REQUESTED** [review]
This is not quite sufficient.

- Some crates still refer to the filecoin repository.
- Some crates have `alpha` in their version.
- Some shared dependencies are workspace dependencies and some are not.

Let's talk more in slack about what is required before we can publish a crate.

File: fil_actor_interface/Cargo.toml (hunk 1 above)
Comment (sudo-shashank) [reply fil_actor_interface/Cargo.toml:3]: I'll mention this in description

**Comment by sudo-shashank** [comment]
> This is not quite sufficient.
> 
> * Some crates still refer to the filecoin repository.
> * Some crates have `alpha` in their version.
> * Some shared dependencies are workspace dependencies and some are not.
> 
> Let's talk more in slack about what is required before we can publish a crate.

added these task in https://github.com/ChainSafe/fil-actor-states/issues/32

This PR only adds all missing package description.

**Review by lemmih — APPROVED** [review]

**Pull request merged**